### PR TITLE
feat: Car image and styles for car tab

### PR DESCRIPTION
**Pull request by thomaskahessay**
Not 100% sure if I did this correctly, but that's what reviews are for 😃

### CODE REVIEW

File: src/pages/Car.tsx [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+import carAerial from '../assets/icons/car-aerial.png';
+import './Car.css';
 
 const Car: React.FC = () => {
     return (
         <IonPage>
             <IonHeader>
-                <IonToolbar>
-                    <IonTitle>Tab 2</IonTitle>
+                <IonToolbar color="#099648" className="CarToolBar">
```
Comment (RatikKapoor) [inline src/pages/Car.tsx:45]: Good job on using Ionic components! 🙌🏽

**Review by elizetran — APPROVED** [review]
🚗 beep beep 🚙

**Review by b4yc — APPROVED** [review]
Looks good :)

**Review by RatikKapoor — CHANGES_REQUESTED** [review]
Will chat in meeting today

**Pull request closed without merging**